libs/cardano-protocol-tpraos/src/Cardano/Protocol/TPraos/BHeader.hs

```diff
@@ -244,73 +249,60 @@ instance Crypto c => DecCBOR (BHBody c) where
     where
       bhBodySize body = 9 + listLenInt (bheaderOCert body) + listLenInt (bprotver body)
 
-data BHeader c = BHeader'
-  { bHeaderBody' :: !(BHBody c)
-  , bHeaderSig' :: !(KES.SignedKES (KES c) (BHBody c))
-  , bHeaderBytes :: BS.ByteString -- Lazy on purpose. Constructed on demand
+data BHeaderRaw c = BHeaderRaw
+  { bhrBody :: !(BHBody c)
+  , bhrSig :: !(KES.SignedKES (KES c) (BHBody c))
```

Comment:
While we are renaming. Getting rid of those ugly abbreviations would also be nice
```suggestion
  , bhrSignature :: !(KES.SignedKES (KES c) (BHBody c))
```